src/util/chains.ts

```diff
@@ -30,11 +30,7 @@ export const V2_SUPPORTED = [
   ChainId.MAINNET,
   ChainId.GOERLI,
   ChainId.SEPOLIA,
-  ChainId.OPTIMISM,
-  ChainId.POLYGON,
-  ChainId.BNB,
-  ChainId.AVALANCHE,
-  ChainId.BASE,
```

Comment:
just want to make sure this is intended

Reply:
Yep this is intended. @adriantrustman will add liquidity on other L2s. Then in routing-api, we also enable there as well https://github.com/Uniswap/smart-order-router/blob/b970aedfec8a9509f9e22f14cc5c11be54d47b35/src/routers/alpha-router/alpha-router.ts#L832